addons/l10n_br_website_sale/views/portal.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="portal_my_details_fields" inherit_id="portal.portal_my_details_fields">
+        <xpath expr="//input[@name='vat']/.." position="before">
+            <t t-if="res_company.country_id.code == 'BR'">
+                <div t-attf-class="mb-1 col-xl-6"/> <!-- Empty div to put the vat and identification type on the same line -->
+
+                <div t-attf-class="mb-1 #{error.get('l10n_ar_afip_responsibility_type_id') and 'o_has_error' or ''} col-xl-6">
+                    <label class="col-form-label" for="l10n_latam_identification_type_id">Identification Type</label>
+                    <t t-if="partner.can_edit_vat()">
+                        <select name="l10n_latam_identification_type_id" t-attf-class="form-select #{error.get('l10n_latam_identification_type_id') and 'is-invalid' or ''}">
+                            <option value="">Identification Type...</option>
+                            <t t-foreach="identification_types or []" t-as="id_type">
```

Comment:
Since for the xpath to be applied we need to install the module l10n_br, some identification type will always be present (since they are added in data) no matter the website so adding a t-if is useless i thinks since it will always be true